array.c

```diff
@@ -6782,13 +6782,18 @@ rb_ary_cycle_size(VALUE self, VALUE args, VALUE eobj)
 
 /*
  *  call-seq:
- *    array.cycle {|element| ... } -> nil
- *    array.cycle(count) {|element| ... } -> nil
- *    array.cycle -> new_enumerator
- *    array.cycle(count) -> new_enumerator
- *
- *  When called with positive Integer argument +count+ and a block,
- *  calls the block with each element, then does so again,
+ *    cycle {|element| ... } -> nil
```

Comment:
I don't think we need this case since the case below encompasses this one.

Reply:
Removed.